atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1373,6 +1373,12 @@ void WebContents::SetSize(const SetSizeParams& params) {
     guest_delegate_->SetSize(params);
 }
 
+void WebContents::SetGuestSize(int width, int height) {
+  SetSizeParams params;
+  params.normal_size.reset(new gfx::Size(width, height));
+  SetSize(params);
```

Comment:
Could we just have `setSize` though?

Calling `setGuestSize(800, 600)` would be the same as calling `setSize({normal: {width: 800, height: 600})` right?